src/pages/index.vue

```diff
@@ -1,44 +1,130 @@
-            </div>
-        </div>
+        <MaCard class="ma-tool-card">
+            <template #default>
+                <div class="ma-tool-content">
+                    <div class="ma-tool-icon">
+                        <MaIcon name="table-list" size="xxlarge" />
+                    </div>
+                    <h3 class="ma-tool-title">Keyword Generator</h3>
+                    <p class="ma-tool-desc">Generate n-gram keywords from text</p>
+                    <MaLinkButton 
+                        href="/keyword-generator"
+                        size="medium"
+                        variant="primary"
+                        :icon="{ name: 'arrow-right', size: 'small' }"
+                        class="ma-tool-button"
+                    >
+                        Open Tool
+                    </MaLinkButton>
+                </div>
+            </template>
+        </MaCard>
+        
+        <MaCard class="ma-tool-card ma-tool-card-placeholder">
+            <template #default>
+                <div class="ma-tool-content">
+                    <div class="ma-tool-icon">
+                        <MaIcon name="chart-bar" size="xxlarge" />
+                    </div>
+                    <h3 class="ma-tool-title">Keyword Density</h3>
+                    <p class="ma-tool-desc">Analyze keyword density in your content</p>
+                    <MaButton 
+                        size="medium"
+                        variant="stroke"
+                        class="ma-tool-button"
+                        disabled
+                    >
+                        Coming Soon
+                    </MaButton>
+                </div>
+            </template>
+        </MaCard>
     </div>
 </template>
 
 <style lang="scss" scoped>
 .ma-home-container {
     display: flex;
-    justify-content: center;
+    flex-direction: column;
     align-items: center;
-    height: 80vh;
-    .ma-card {
-        border: 1px solid #ccc;
-        border-radius: 0.25rem;
-        padding: 2rem;
-        text-align: center;
-        .ma-card-text {
-            margin-bottom: 2rem;
+    gap: 2rem;
+    padding: 3rem 1rem;
+    min-height: 80vh;
+    background: linear-gradient(135deg, #f5f7fa 0%, #c3cfe2 100%);
+}
+
+.ma-tool-card {
+    width: 100%;
+    max-width: 400px;
+    border-radius: 16px;
+    box-shadow: 0 8px 32px rgba(0, 0, 0, 0.1);
+    transition: all 0.3s ease;
+    
+    &:hover {
+        transform: translateY(-4px);
+        box-shadow: 0 16px 48px rgba(0, 0, 0, 0.15);
+    }
+    
+    &.ma-tool-card-placeholder {
+        opacity: 0.7;
+        
+        &:hover {
+            transform: translateY(-2px);
+            opacity: 0.8;
         }
     }
 }
 
-.ma-btn {
-    padding: 0.5rem 1rem;
+.ma-tool-content {
+    text-align: center;
+    padding: 2rem;
+}
+
+.ma-tool-icon {
+    margin-bottom: 1.5rem;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    color: #14b8a6;
+}
```

Comment:
```suggestion
.ma-tool-content {
    text-align: center;
    padding: 2rem;
    display: flex;
    flex-direction: column;
    justify-content: center;
    align-items: center;
}

.ma-tool-icon {
    margin-bottom: 1.5rem;
    color: #14b8a6;
}
```

I explained in the template, please check your whole project, maybe you have unnecessary divs, and you should apply the flexbox css's into your container div, then everything will be placed in right places, in this case now we give the flexbox css's into parent div. Please check your usage in the project.